src/celeritas/phys/FourVector.hh

```diff
@@ -62,4 +64,26 @@ inline CELER_FUNCTION void boost(Real3 const& v, FourVector* p)
     p->energy = gamma * (p->energy + vp);
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Add two four-vectors.
+ */
+inline CELER_FUNCTION FourVector add(FourVector const& a, FourVector const& b)
```

Comment:
Okay, I added operator+ and the assignment operator+=.  I also realize that it may be better to template FourVector with the precision type, which I may do in another MR.  I did not add other assignment/arithmetic operators as they are not likely used.